packages/rs-drive/src/drive/verify/mod.rs

```diff
@@ -15,3 +15,5 @@ pub mod state_transition;
 
 /// Represents the root hash of the grovedb tree
 pub type RootHash = [u8; 32];
+/// Simply a collection of 32 Bytes. It's just that simple
+pub type Bytes32 = [u8; 32];
```

Comment:
No, don't do this.

Reply:
2899837cca0a8214ef0ed12723cbf16e8ce85f53